src/compiler/call.ts

```diff
@@ -916,14 +938,45 @@ export function compileCallExpression(
 		}
 
 		let callPath = compileExpression(state, exp);
-		if (
+
+		if (node.hasQuestionDotToken()) {
+			assertUnquestionable(node);
```

Comment:
I'm confused about this part. Are you adding `?.` logic? This seems to always error.

Reply:
It is supposed to always error. I was experimenting with `?.` logic, but found it to be too incompatible with the old system, so I just put the error back without deleting the unnecessary logic. I didn't think it would matter to much if it sits there, since we're planning on replacing this code with the refactor.

Reply:
I'd prefer we remove it. It makes it more clear what this PR is doing.